server/delay_server/src/irodsDelayServer.cpp

```diff
@@ -667,8 +669,10 @@ int main(int argc, char** argv)
         try {
             return irods::get_advanced_setting<const int>(irods::CFG_NUMBER_OF_CONCURRENT_DELAY_RULE_EXECUTORS);
         }
-        catch (const irods::exception& e) {
-            logger::delay_server::error(e.what());
+        catch (...) {
+            logger::delay_server::warn("Could not retrieve number of concurrent delay rule executors from configuration. "
```

Comment:
consider showing the actual entry name that was missing - so the admin can just copy/paste to fix?

Reply:
done